docs/source/releasing.rst

```diff
@@ -37,31 +37,18 @@ Things to do for releasing:
 
 * create pull request to ``python-trio/trio``'s "master" branch
 
-* announce PR on gitter
-
-   + wait for feedback
-
-   + fix problems, if any
-
 * verify that all checks succeeded
 
-* acknowledge the release PR
-
-   + or rather, somebody else should do that
-
-* tag with vVERSION
+* tag with vVERSION, push tag
 
 * push to PyPI
 
    + ``python3 setup.py sdist bdist_wheel upload``
 
-* announce on gitter
-
-* update version number
+* update version number in the same pull request
 
    + add ``+dev`` tag to the end
 
-* prepare another pull request to "master"
-
-   + acknowledge it
+* approve and merge the release pull request
```

Comment:
You can post an approving review if you want, but I don't know if it actually does anything :-). We don't have github set to enforce approvals before merge. (Actually, now that I think about it, I think you may not even be able to post an approval on your own PR?)

Reply:
Right, we can't approve our own PRs, but we can merge them, fixed